src/angular/components/send/send.component.ts

```diff
@@ -85,6 +85,9 @@ export class SendComponent implements OnInit {
         this.loading = true;
         const sends = await this.sendService.getAllDecrypted();
         this.sends = sends;
+        if (this.onSuccessfulLoad() != null) {
```

Comment:
This will execute `this.onSuccessfulLoad()` and evaluate the result (`!= null`), not sure that's intended since you're executing it again below. I believe you need to remove the `()` parens at the end to test if the delegate/method is `null` vs. the result of evaluation/execution of the same.

Reply:
Great catch! Updated this delegate and the others as well.